### ISSUE TITLE
Title bar on the side?

### ISSUE THREAD

**Issue by Vermoot** [paulmcauley/klassy#48]
I like having my title bars on the side of the window rather than the top (less wasted space on a landscape screen).

Is that possible at all with Klassy, or could that be added as a feature?

**Comment by nclarius** [comment]
Or the bottom. I have my panel at the top, so it would make sense to have the title bar at the opposite side so I could easily hit the title bar/buttons  by moving my mouse against the screen edge/corners.

**Comment by paulmcauley** [comment]
It is possible to do, but it would be a lot of work...

**Comment by developer91234** [comment]
Don't see the use case for this, since it would essentially prevent access to the titlebar on floating window managers when a portion of the window is off screen ... which happens pretty much all the time.

I do see a use for case for this however, as well for for decorations that can shapeshift and do more stuff generally:

![image](https://github.com/paulmcauley/klassy/assets/43324854/76813622-0f78-4e50-9418-a484d24f9c81)

Among other things, this would allow you to force server-side decorations for csd apps and make it completely seamless. Or to save space when using traditional apps.

Lotta work of course but would be a genuine breakthough.

@paulmcauley

**Comment by nclarius** [comment]
> it would essentially prevent access to the titlebar on floating window managers when a portion of the window is off screen 

No, there is no reason why this would be any different from having the titlebar at the top of the screen. When dragged by the titlebar (which is what most people do) you naturally won't be able to drag it further than that anyway, and when you do it by other means (e.g. meta + mouse anywhere inside the window) you'll also be able to get it back that way.
